spec/requests/api/v2.0/applications_spec.rb

```diff
@@ -9,9 +9,8 @@
 
   let(:headers)          { {"CONTENT_TYPE" => "application/json", "x-rh-identity" => identity} }
   let(:collection_path)  { "/api/v2.0/applications" }
-  let(:source)           { Source.create!(:source_type => source_type, :tenant => tenant, :uid => SecureRandom.uuid, :name => "my-source") }
-  let(:source_type)      { SourceType.create!(:name => "my-source-type", :product_name => "My Source Type", :vendor => "ACME") }
-  let(:application_type) { ApplicationType.create!("name" => "my-application", :supported_source_types => ["my-source-type"]) }
+  let(:source)           { create(:source, tenant: tenant, uid: SecureRandom.uuid) }
```

Comment:
Same, about tenant here.

Reply:
Same here. However, it's probably not needed for every context. I just used the explicit tenant everywhere to be safe.